packages/scan/project.json

```
@@ -0,0 +1,40 @@
+{
+  "root": "packages/scan",
+  "sourceRoot": "packages/scan/src",
+  "projectType": "library",
+  "targets": {
+    "build": {
+      "executor": "@nrwl/js:tsc",
+      "outputs": ["{options.outputPath}"],
+      "options": {
+        "outputPath": "dist/packages/scan",
+        "tsConfig": "packages/scan/tsconfig.lib.json",
+        "packageJson": "packages/scan/package.json",
+        "main": "packages/scan/src/index.ts",
+        "assets": ["packages/scan/*.md", "LICENSE"]
```

It won't work as expected. See how it is done in other modules
```suggestion
        "assets": [
          "packages/scan/*.md",
          {
            "glob": "LICENSE",
            "input": "",
            "output": ""
          }
        ]
```